boot/bootutil/src/encrypt_ec256.c

```diff
@@ -0,0 +1,144 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2018-2019 JUUL Labs
+ * Copyright (c) 2019-2021 Arm Limited
+ */
+
```

Comment:
Please include the mcuboot_config.h first - same for the rest of the `encrypted_<alg>.c` files.

Reply:
Done